src/Spine/index.js

```diff
@@ -1,10 +1,7 @@
-var PIXI = require('pixi.js'),
-    spine = require('./SpineRuntime');
-
+var PIXI = require('pixi.js');
+var spine = require('../SpineRuntime');
```

Comment:
seems weird for the spine runtime to be in a cap-first folder, when it isn't a class file. I recommend changing the runtime's folder to simply `runtime/`